core/src/wsv.rs

```diff
@@ -103,16 +109,47 @@ impl<W: WorldTrait> WorldStateView<W> {
             config: Configuration::default(),
             transactions: DashSet::new(),
             blocks: Arc::new(Chain::new()),
+            metrics: Arc::new(Metrics::default()),
         }
     }
 
-    /// [`WorldStateView`] constructor with configuration.
+    #[cfg(test)]
+    pub fn transactions_number(&self) -> u64 {
+        self.blocks.iter().fold(0_u64, |acc, block| {
+            acc + block.as_inner_v1().transactions.len() as u64
+                + block.as_inner_v1().rejected_transactions.len() as u64
+        })
+    }
+
+    /// Returns [`Some`] milliseconds since the genesis block was
+    /// committed, or [`None`] if it wasn't.
+    pub fn genesis_timestamp(&self) -> Option<u128> {
+        self.blocks
+            .iter()
+            .next()
+            .map(|val| val.as_inner_v1().header.timestamp)
+    }
```

Comment:
Genesis block was created at this time, we don't know exactly when it was committed. But I guess this doesn't matter much for TPS metric.

Reply:
I will change the description of this metric to align with what is actually measured.